eogrow/utils/testing.py

```diff
@@ -266,3 +264,31 @@ def create_folder_dict(config_folder: str, stats_folder: str, subfolder: Optiona
         "config_folder": os.path.join(config_folder, subfolder) if subfolder else config_folder,
         "stats_folder": os.path.join(stats_folder, subfolder) if subfolder else config_folder,
     }
+
+
+def run_dummy_data_pipeline(
+    config_file: str, *, config_folder: str, folder_key: Optional[str] = None, reset_folder: bool = True, **_: Any
+) -> None:
+    """A default way of testing a pipeline
+
+    :param config_file: Name of the config file containing a dummy data generating pipeline
+    :param config_folder: A path to folder containing the config file
+    :param folder_key: Folder containing results of the pipeline, if missing it's inferred from config
+    :param reset_folder: If True it will delete content of the folder with results before running the pipeline
+    """
+    if not config_file.endswith(".json"):
+        config_file += ".json"
```

Comment:
Perhaps we could even raise an error in case the name doesn't end with `".json"`? Automatic adding of file format can produce weird file names.